dashboard/app/views/layouts/_http_support.html.haml

```diff
@@ -9,18 +9,30 @@
     %i.fa.fa-warning
     = t("compatability.https.header")
   :javascript
+    function setCookie(key, value, days) {
```

Comment:
Can we use jQuery's `$.cookie` instead of manually manipulating `document.cookie`?